.circleci/config.yml

```diff
@@ -126,6 +175,9 @@ workflows:
       - test_ubuntu_latest
       - test_ubuntu_rolling
       - test_archlinux
+      - test_ubuntu_latest_opencv
+      - test_ubuntu_rolling_opencv
+      - test_archlinux_opencv
```

Comment:
What about instead of installing the system all over, we would just test it without opencv, than install opencv to the same system and then test it again? (i.e., just adding two steps to the old `test_ubuntu_latest` etc.)